manifests/08-prometheus_rule.yaml

```
@@ -20,4 +20,13 @@ spec:
       for: 5m
       labels:
         severity: info
+    - alert: SCANotAvailable
+      annotations:
+        description: 'Simple content access is not enabled. See https://access.redhat.com/articles/simple-content-access to enable it.'
+        summary: Simple content access certificates are not available.
+      expr: |
+       cluster_operator_conditions{name="insights", condition="SCANotAvailable", reason="NotFound"} == 1
```

To avoid alert resets with failed scrapes, it's recommended to use an *_over_time() function (max_over_time() in this case):

```suggestion
       max_over_time(cluster_operator_conditions{name="insights", condition="SCANotAvailable", reason="NotFound"}[5m]) == 1
```